### ISSUE TITLE
Duplicated footnotes

### ISSUE THREAD

**Issue by anermina** [metanorma/metanorma-itu#135]
Footnotes which are referenced more than once are numbered differently and described each time.

original:
![image](https://user-images.githubusercontent.com/55683568/69383919-3bce1280-0cbb-11ea-866d-dd0338e89e32.png)

rendered:
![image](https://user-images.githubusercontent.com/55683568/69383943-48eb0180-0cbb-11ea-8dcb-ae1dca5fddef.png)

[footnotes_res.zip](https://github.com/metanorma/metanorma-itu/files/3877149/footnotes_res.zip)

**Comment by ronaldtse** [comment]
A similar issue also shows up in the ISO Rice document where footnote references are reused via “footnoteref”, which triggers a deprecation warning in asciidoctor.

**Comment by opoudjis** [comment]
Yes. This can go both ways: Some specs want repetition of the footnote, some do not. Word's default is not to, and I recall that Word does not like the reuse of footnote references; in fact, I have removed the reuse functionality recently, and I'll need to look up why I did.

footnoteref does indeed seem to have been deprecated between Asciidoctor 1 and Asciidoctor 2, but that is an Asciidoctor markup issue.

**Comment by ronaldtse** [comment]
> What is happening is simply that the Asciidoctor markup of the document needs to signal explicitly that the footnote is to be reused; @anermina, you have not done so yet. I also need to make sure that the new method for signalling that in Asciidoctor, footnote:id[], is recognised in XML generation. (The "id" is to be replaced by an arbitrary string, which needs to be the same for the two footnote instances.) Once that is done, I'll remove all instances of footnoteref from our repositories.

@opoudjis this should probably be a new ticket for metanorma-standoc. Will make one for that.

**Comment by anermina** [comment]
> What is happening is simply that the Asciidoctor markup of the document needs to signal explicitly that the footnote is to be reused; @anermina, you have not done so yet. I also need to make sure that the new method for signalling that in Asciidoctor, `footnote:id[]`, is recognised in XML generation. (The "id" is to be replaced by an arbitrary string, which needs to be the same for the two footnote instances.) Once that is done, I'll remove all instances of footnoteref from our repositories.

@opoudjis When the footnote first appeared, it was indicated by following: 
{blank}footnote:appointing[In special cases, WTSA may appoint the chairman and request the Radiocommunication Assembly to appoint a vice-chairman.]

When it appeared again, it was indicated by following:
{blank}footnote:appointing[]

I don't understand what should be changed... Can you please let me know what is the proper markup?

**Comment by opoudjis** [comment]
@anermina This functionality was implemented by the older footnoteref macro, and I think it is not recognising the footnote:id[] markup. Since footnoteref is deprecated, don't change anything: I will need to update the code to recognise the current macro instead.

**Comment by opoudjis** [comment]
Oh, I see. Repeating footnotes are suppressed in HTML, but not in DOC; the change in markup has not affect that.

I removed that behaviour in https://github.com/metanorma/isodoc/issues/86. I don't have detailed notes about what triggered the concern in the NIST document I was editing, but I can see things going back in Word quite easily with the suppression of expected behaviour. OTOH, repeating footnotes is permitted behaviour in Word: https://support.office.com/en-us/article/refer-to-the-same-footnote-or-endnote-more-than-once-9925ad08-2593-49ac-842b-c0f55b94e137

Having exported that functionality to HTML, I see that my previous implementation was naive, and this implementation is going to require inserting bookmarks, and will be involved:

* Wrap the first instance of the footnote in Word with `<span
style='mso-bookmark:_Ref26828031'>`
* Make the co-referential instances of the footnote in Word be `<span
style='mso-element:field-begin'></span> NOTEREF _Ref26828031 \f \h <span
style='mso-element:field-separator'></span><span
class=MsoFootnoteReference>1</span><span
style='mso-element:field-end'></span>`

I will do it, but not tonight. This has already taken me a while to look into, and there are more urgent issues.
